mobile-app/app/screens/AppNavigator/screens/Auctions/components/ManageBids.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react'
+import { tailwind } from '@tailwind'
+import { ThemedScrollView } from '@components/themed'
+import { useFeatureFlagContext } from '@contexts/FeatureFlagContext'
+import { View } from 'react-native'
+import { InfoText } from '@components/InfoText'
+import { translate } from '@translations'
+import { useDispatch, useSelector } from 'react-redux'
+import { RootState } from '@store'
+import { useEffect } from 'react'
+import { fetchVaults } from '@store/loans'
+import { useWhaleApiClient } from '@shared-contexts/WhaleContext'
+import { useWalletContext } from '@shared-contexts/WalletContext'
+import { LoanVaultLiquidated, LoanVaultLiquidationBatch } from '@defichain/whale-api-client/dist/api/loan'
+import { BidCard } from './BidCard'
+import { EmptyBidsScreen } from './EmptyBidsScreen'
+
+export function ManageBids (): JSX.Element {
```

Comment:
If we can add a loader here it would be better since there's already one for Browse Auctions

Reply:
Noted, Will create a different loader for bids

Reply:
let's close this as we wont have `manage bids` section the next release